src/layouts/shortcodes/cloud-prepare/openstack.md

```diff
@@ -0,0 +1,37 @@
+Prepare OpenShift-on-AWS **cluster-a** for Submariner:
+
+<div class="notices note">
+<p>The default EC2 instance type for the Submariner gateway node is <code>c5d.large</code>,
+optimized for better CPU which is found to be a bottleneck for IPsec and Wireguard drivers.
+Alternatively, you can choose to deploy using a different instance type.</p>
+</div>
+
+```bash
+export KUBECONFIG=cluster-a/auth/kubeconfig
+subctl cloud prepare aws --ocp-metadata path/to/cluster-a/metadata.json {{ with $.Get "nattPort" }}--natt-port {{.}}{{ end }}
+```
+
+Prepare OpenShift-on-OpenStack **cluster-b** for Submariner:
+
+The default OpenStack vm instance type for the Submariner gateway node is <code>PnTAE.CPU_4_Memory_8192_Disk_50</code>,
+Alternatively, you can choose to deploy using a different instance type.
+You need to use the appropriate cloud name from clouds.yaml, here it uses **OpenStack**
+
+```bash
+export KUBECONFIG=cluster-b/auth/kubeconfig
+subctl cloud prepare rhos --ocp-metadata path/to/cluster-b/metadata.json --cloud-entry\
+openstack {{ with $.Get "nattPort" }}--natt-port {{.}}{{ end }}
```

Comment:
The "\" at the end is not getting rendered properly. We can either remove it or include an extra space so that users can directly copy-paste the command.

Reply:
is it the natport not getting rendered correctly?

Reply:
the natport itself is not displayed, please take a look.

Reply:
Done.